src/test/utils/DSTestPlus.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+
+import {DSTest} from "@ds/test.sol";
+import {ERC20} from "@solmate/tokens/ERC20.sol";
+
+import {stdCheats, stdError} from "@std/stdlib.sol";
+import {Vm} from "@std/Vm.sol";
+
+contract DSTestPlus is DSTest, stdCheats {
+
+    /// @dev Use forge-std Vm logic
+    Vm public constant vm = Vm(HEVM_ADDRESS);
+
+    function assertERC20Eq(ERC20 erc1, ERC20 erc2) internal {
+        assertEq(address(erc1), address(erc2));
+    }
```

Comment:
Seems this assertion helper isn't used anywhere so I think we can delete this function, the `ERC20` import, and the full somate dependency. And since we don't use `forge-std` anywhere except for the `Vm` interface (unless I missed it) maybe we just inline that interface within this file to reduce deps